src/main/resources/templates/start.html

```diff
@@ -29,12 +29,12 @@ <h2>Collections</h2>
     </div>
     <!-- Items of a selected collection section-->
     <div class="collection-div">
-        <div class="container item-div">
+        <div class="container pt-3 pb-3">
```

Comment:
Same as the other padding change. Wanted to utilize the existing Bootstrap classes to guarantee that it will work.

Citation: https://getbootstrap.com/docs/4.0/utilities/spacing/